app/Http/Controllers/MyGalleryController.php

```diff
@@ -109,7 +109,7 @@ public function destroy(Gallery $gallery): RedirectResponse
     {
         $this->authorize('delete', $gallery);
 
-        $gallery->delete();
+        $gallery->forceDelete();
```

Comment:
since gallery  now has a deleted_at column it need to be forced deleted when deleted by the user